ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -256,7 +256,7 @@ private async Task ImportReferenceLabResultsAsync(IList<Notification> notificati
             foreach (var (legacyId, referenceLaboratoryNumber) in matches)
             {
                 var notificationId = notifications.Single(n => n.ETSID == legacyId).NotificationId;
-                var success = await _specimenService.MatchSpecimenAsync(notificationId, referenceLaboratoryNumber, "SYSTEM");
+                var success = await _specimenService.MatchSpecimenAsync(notificationId, referenceLaboratoryNumber, "SYSTEM", true);
```

Comment:
The context is a bit difficult to grab from just the boolean value here, so perhaps it's worth adding the parameter name?
```suggestion
                var success = await _specimenService.MatchSpecimenAsync(notificationId, referenceLaboratoryNumber, "SYSTEM", isMigrating: true);
```